### PR TITLE
Check if campaign exists, belongs to account and has content

### PR DESCRIPTION
**Pull request by maurocardinali**
This PR validates if

- Campaign exists
- Campaign belongs to account
- Campaign has content.

If doesn't exists or belongs to a different user, repository returns null.
If hasn't any content, repository returns an empty contentRow.
If none of these scenarios occurs, repository returns query result.

### CODE REVIEW

File: Doppler.HtmlEditorApi/Infrastructure/Repository.cs [hunk 1]
```diff
@@ -19,15 +19,41 @@ public async Task<ContentRow> GetCampaignModel(string accountName, int campaignI
...
+
+                var queryResult = await connection.QueryFirstOrDefaultAsync<dynamic>(databaseQuery, new { IdCampaign = campaignId, accountName });
+
+                if (!queryResult.CampaignBelongsUser || !queryResult.CampaignExists)
+                {
+                    return null;
+                }
+                else
+                if (!queryResult.CampaignHasContent)
+                {
```
Comment (andresmoschini) [inline Doppler.HtmlEditorApi/Infrastructure/Repository.cs:40]:
```suggestion
                }
                else if (!queryResult.CampaignHasContent)
                {
```

Or simply:

```suggestion
                }
                
                if (!queryResult.CampaignHasContent)
                {
```

File: Doppler.HtmlEditorApi/Infrastructure/Repository.cs [hunk 2]
```diff
@@ -19,15 +19,41 @@ public async Task<ContentRow> GetCampaignModel(string accountName, int campaignI
...
+                    Content = queryResult.Content,
+                    Meta = queryResult.Meta,
+                    EditorType = queryResult.EditorType,
+                    IdCampaign = queryResult.IdCampaign
+                };
             }
         }
 
+
+
```
Comment (andresmoschini) [inline Doppler.HtmlEditorApi/Infrastructure/Repository.cs:56]:
```suggestion
```

File: Doppler.HtmlEditorApi/Infrastructure/Repository.cs (hunk 1 above)
Comment (maurocardinali) [reply Doppler.HtmlEditorApi/Infrastructure/Repository.cs]: Good. Fixed

File: Doppler.HtmlEditorApi/Infrastructure/Repository.cs (hunk 2 above)
Comment (maurocardinali) [reply Doppler.HtmlEditorApi/Infrastructure/Repository.cs]: REsolved!

**Comment by andresmoschini** [comment]
👍

**Pull request merged**